server/src/main/java/org/apache/druid/server/log/LoggingRequestLogger.java

```diff
@@ -68,7 +68,6 @@ public void logNativeQuery(RequestLogLine requestLogLine) throws IOException
           MDC.put("hasFilters", Boolean.toString(query.hasFilters()));
           MDC.put("remoteAddr", requestLogLine.getRemoteAddr());
           MDC.put("duration", query.getDuration().toString());
-          MDC.put("descending", Boolean.toString(query.isDescending()));
```

Comment:
Don't we want to log this if the query is TImeseriesQuery?

Reply:
oh, i guess we could, not sure how interesting it really is though which is why i removed it